examples/domains.ts

```diff
@@ -210,7 +205,7 @@ mySdk.domains
     domainId,
     "<roleName>",
     { offset: 0, limit: 10 },
-    token
+    token,
```

Comment:
Remove this one as well and other places where lint didn't pick up
```suggestion
    token
```